### PR TITLE
PE-D Bug Fixing 1

### LINKED ISSUE #78: [PE-D][Tester C] updateBudget command does not accept input of type DOUBLE
Steps to reproduce:
1. Launch the application.
2. Enter your name.
3. Enter 'updateBudget 99.99'
4. Error message is shown.

Expected behaviour:
Budget is updated to 99.99

Actual behaviour: 
Error message is shown and budget is not updated.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: lckjosh/ped#1

### LINKED ISSUE #84: [PE-D][Tester C] Format of addExpense command in UG shows addIncome
The format shows addIncome instead of addExpense. Perhaps it was a typo that needs fixing. 

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: lckjosh/ped#3

### LINKED ISSUE #86: [PE-D][Tester E] Must enter twice to execute deleteBudget
Steps to reproduce:
1. Have no budget updated 
2. Enter "deleteBudget 1"

Expected: 
Command executes immediately

Actual:
I have to press enter again, then the command executes. Else it stays blank.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: yingx9/ped#5

### LINKED ISSUE #96: [PE-D][Tester E] Crashes when entering blank input
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
Steps to reproduce:

1. Run jar file
2. Enter name 
3. Enter a blank space " " 

Expected:

Caught error stating no blank input or to enter a command

Actual:

Program crashes with exception

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: yingx9/ped#9

### LINKED ISSUE #99: [PE-D][Tester C] Entering deleteBudget when there is no budget does not show error message.
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
When there is no budget, entering the deleteBudget command does not provide any output to the user at all.

Steps to reproduce:
1. Launch the program and enter name.
2. Enter 'deleteBudget'
3. There is no output shown.

Expected behaviour:
An error message stating that there is no budget to delete. 

Actual behaviour:
No output shown

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: lckjosh/ped#6

### LINKED ISSUE #100: [PE-D][Tester A] Double is not accepted when initial setting of budget
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
Doubles are not accepted even though the format says so

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: NeoMinWei/ped#10

### LINKED ISSUE #104: [PE-D][Tester A] CashLeh visual seems to be formatted wrongly
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
The top ends of the visual are not connected

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: NeoMinWei/ped#13

### LINKED ISSUE #105: [PE-D][Tester B] Instructions for opening jar seems to be wrong
**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: SebasFok/ped#5

### PR DESCRIPTION
**Pull request by danielpappa**
closes #84
closes #86
closes #96 
closes #99 
closes #100
closes #104 
closes #105

### CODE REVIEW

File: docs/UserGuide.md
Comment (mcmc101001) [inline docs/UserGuide.md:1]: should this be `java -jar CashLeh.jar ` instead?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Example of usage:
 ### Adding an expense: `addExpense`
 Adds an expense with a description, amount, date and category.
 
-Format: `addIncome DESCRIPTION /amt AMOUNT /date DATE /cat CATEGORY`
+Format: `addExpense DESCRIPTION /amt AMOUNT /date DATE /cat CATEGORY`
```
Comment (mcmc101001) [inline docs/UserGuide.md:64]: good catch!

**Review by mcmc101001 — APPROVED** [review]
LGTM

**Review by mcmc101001 — CHANGES_REQUESTED** [review]
Modify UG to `java -jar CashLeh.jar `

**Review by BenedictChannn — APPROVED** [review]
LGTM

**Review by mcmc101001 — APPROVED** [review]
LGTM

**Pull request merged**